readme.md

```diff
@@ -19,6 +19,7 @@ Návod je určený pro Windows, v Linuxu by mělo stačit nainstalovat všechno
   - PHP > PHP settings > upload_max_filesize > vybrat 8M – navýšit limit post souboru
   - Apache > Apache modules > zaškrtnout expires_module – zapnout mod_expires
   - MySQL > MySQL settings > sql-mode > none – hlavně kvůli povolení negroupnutých sloupců při GROUP BY a doplňování nulových hodnot i do sloupců, co nemají default
+  - MySQL > my.ini - asi na padesátém řádku nastavit `default-storage-engine=InnoDB` (jinak neprobíhá na některých strojích migrace databáze korektně)
```

Comment:
Dobrý. Ale mohlo by být lepší.-)
1. Řádek se může dost lišit, navíc ta direktiva nemusí být v my.ini vůbec

Upravil bych to na něco jako "V my.ini najít a nastavit `default-storage-engine=InnoDB` (pokud nic takového v my.ini nemáš, tak to dej na samostatný řádek hned pod `[mysqld]`)".

2. To že migrace neprobíhá je dobré info, ale osobně mi chybí "proč se to pokazí". Přidal bych edukativní "jinak neprobíhá na některých strojích migrace databáze korektně, protože bez InnoDB nejde provázat tabulky cizími klíči", nebo tak něco

Reply:
V pořádku, když už se něco dělá, tak nejlíp jak to jde! Souhlas.
Dnes pošlu aktualizované.